### ISSUE TITLE
A11y process documentation request from OCTO

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#63109]
## About your team

- Team name: Accessibility in OCTO
-	OCTO-DE product owner: @artsymartha68 
-	Product manager: @artsymartha68 

 
## CAIA Support Request

Now that @eli-oat is on the team, it would be great to have him tackle some of the work that has been rumbling in my head. This request involves documenting CAIA a11y testing/auditing/working/etc processes so that new folks can get up to speed and we can set some standards around the a11y work. 

Content has had a few years ahead of a11y and now we need to catch up!

This could include:

- a11y tooling
- a11y audit documentation (something that was brought up today at the a11y CoP meeting)
- creating artifacts of the a11y work so that a future team can know the status of the a11y of the product
- a generic timeline of how to work with a team when an a11y request comes in - collab cycle milestones of course, but there is probably more! The recent kickoff meeting is a recent addition, there are probably more.
- working with researchers, designers, developers, etc.
- moving doc out of Google into GitHub (or Confluence?)
- and much much more I'm sure

### Which are you doing?
- [x] **Launching a new** page, tool, form, or app
- [ ] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [x] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe

No deadline, this is a lot so will be done in stages via various sub-tickets, I would assume

### What's your team’s next step and its timing?

- **Our next step is** ....
- **Provide date if available**: this quarter

#### Timing for your next step: 
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [x] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**:
- **User flows**:
- **Prototype or mockups**:
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [x] Martha (IT ME!)

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**
